build.gradle

```diff
@@ -1,15 +1,33 @@
+buildscript {
+    repositories { mavenCentral() }
+    dependencies {
+        classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.5.3'
+    }
+}
+
+plugins {
+    id 'com.bmuschko.nexus' version '2.3.1'
+    id 'pl.allegro.tech.build.axion-release' version '1.4.0'
+}
+
 apply plugin: 'java'
 apply plugin: 'groovy'
 apply plugin: 'maven-publish'
```

Comment:
we don't need this - it's `maven` that handles the publication (attached by nexus plugin)